community/pike/APKBUILD

```diff
@@ -0,0 +1,235 @@
+			$pkgname-pcre \
+			$pkgname-sdl \
+			$pkgname-svg \
+			$pkgname-sqlite \
+			$pkgname-dev \
+			$pkgname-doc \
+			$pkgname-core"
+source="http://pike.lysator.liu.se/pub/pike/all/${pkgver}/Pike-v${pkgver}.tar.gz"
+builddir="$srcdir/"
+
+unpack() {
+	cd "$builddir"
+	tar xzf Pike-v${pkgver}.tar.gz --strip-components=1
+}
+
+build() {
+	cd "$builddir"
+	make CONFIGUREARGS=" \
+		--prefix=/usr \
+		--disable-make_conf \
+		--disable-noopty-retry \
+		--without-cdebug \
+		--without-bundles \
+		--without-ssleay \
+		--with-gmp \
+		--with-crypt \
+		--with-bignums \
+		--without-rtldebug \
+		--with-Bz2 \
+		--without-fftw \
+		--without-_Ffmpeg \
+		--with-gdbm \
+		--without-GTK1 \
+		--with-GTK2 \
+		--with-jpeglib \
+		--without-Kerberos \
+		--without-msql \
+		--without-mysql \
+		--without-Odbc \
+		--without-oracle \
+		--with-GL \
+		--with-GLUT \
+		--with-_Regexp_PCRE \
+		--without-libpdf \
+		--without-sass \
+		--with-sane \
+		--with-SDL \
+		--with-SDL_Mixer \
+		--with-svg \
+		--with-tifflib \
+		--without-freetype \
+		--without-vcdiff \
+		--with-_Image_WebP \
+		--with-zlib"
+}
+
+package() {
+	cd "$builddir"
+	depends=""
+	for pkg in $(printf "%s" "$subpackages" | tr ' ' '\n'); do
+		depends="$depends $pkg=$pkgver-r$pkgrel"
+	done
+	make INSTALLARGS="--traditional" buildroot="$pkgdir" install_nodoc
+}
+
+_move_item() {
+	local bn=$(basename "$1")
+	local dn=$(dirname "$1")
+	local dst="$2/$dn"
+	mkdir -p "$dst"
+	mv "$1" "$dst/$bn"
+}
+
+_install() {
+	local bn=$(basename "$1")
+	local dn=$(dirname "$1")
+	for item in `find $dn -maxdepth 1 -name "$bn"`; do
+		_move_item "$item" "$2"
+	done
+}
+
+bzip2() {
+	cd "$pkgdir"
+	pkgdesc="$pkgname bzip2 support"
+	depends="pike-core=$pkgver-r$pkgrel bzip2"
+	_install "usr/lib/pike/modules/*Bz2.*" "$subpkgdir"
+}
+
+dnssd() {
+	cd "$pkgdir"
+	pkgdesc="$pkgname DNS SD support"
+	depends="pike-core=$pkgver-r$pkgrel"
+	_install "usr/lib/pike/modules/*DNS_SD.*" "$subpkgdir"
+}
+
+fuse() {
+	cd "$pkgdir"
+	pkgdesc="$pkgname fUSE support"
+	depends="pike-core=$pkgver-r$pkgrel"
+	_install "usr/lib/pike/modules/*Fuse.so" "$subpkgdir"
+	_install "usr/lib/pike/modules/Fuse.pmod" "$subpkgdir"
+}
+
+gdbm() {
+	cd "$pkgdir"
+	pkgdesc="$pkgname GDBM support"
+	depends="pike-core=$pkgver-r$pkgrel gdbm"
+	_install "usr/lib/pike/modules/*Gdbm.*" "$subpkgdir"
+}
+
+gl() {
+	cd "$pkgdir"
+	pkgdesc="$pkgname OpenGL/GLUT support"
+	depends="pike-core=$pkgver-r$pkgrel freeglut mesa-gl libsm"
+	_install "usr/lib/pike/modules/*GL*" "$subpkgdir"
+}
+
+gtk() {
+	cd "$pkgdir"
+	pkgdesc="$pkgname GTK2 support"
+	depends="pike-image=$pkgver-r$pkgrel libglade gtk+2.0"
+	_install "usr/lib/pike/modules/*GTK2*" "$subpkgdir"
+	_install "usr/lib/pike/modules/Tools.pmod/PV.pike" "$subpkgdir"
+}
+
+image() {
+	cd "$pkgdir"
+	pkgdesc="$pkgname image handling modules"
+	depends="pike-core=$pkgver-r$pkgrel giflib-utils libjpeg tiff libwebp"
+	_install "usr/lib/pike/?.?/modules/Image.pmod" "$subpkgdir"
```

Comment:
With the quotes, the first argument to `_install` is likely not doing what you expect.